### DISCUSSION TITLE
Apple-IIe?

### DISCUSSION THREAD

**Discussion by vitasam** [markadev/AppleII-VGA#50] [General]
Hi, maybe it was discussed already - will that card work for Apple-IIe (especially in 80-column mode)?

**Comment by markadev** [comment]
Yup, all the Apple IIe modes are supported when the IIe firmware is used.

**Reply by vitasam** [reply]
Great! What about additional 64KB or memory, available for those 80-columns IIe cards?

I'm just considering to buy Apple-IIe and thinking what could be the best option to get VGA output

**Reply by markadev** [reply]
Hey there, I'm not sure what you mean.  If you have an Apple IIe then you'll probably also want to have an 80 column card installed in it.  The "80 column card" for a IIe is really just a 64KB memory expansion, some of which is used for the IIe video modes.

**Reply by vitasam** [reply]
Indeed I did not know it, I thought that "80-column card" is actually an external video-card with additional 64KB memory. This is what I was told in the email: "He has a //e with 80 column card (and which includes 64KB extra RAM (a standard card most //e have installed))"
